tests/installation/partitioning_filesystem.pm

```diff
@@ -22,7 +22,7 @@ sub run {
     assert_screen 'edit-proposal-settings';
     wait_screen_change { send_key $cmd{guidedsetup} };
 
-    select_first_hard_disk if (check_screen('select-hard-disks', 0) && check_var('BACKEND', 'ipmi'));
+    select_first_hard_disk if (check_screen('select-hard-disks', 5) && check_var('BACKEND', 'ipmi'));
```

Comment:
please don't

Reply:
Agreed to pull out this for a better approach in future.